content/cpp/concepts/math-functions/terms/nearbyint/nearbyint.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'nearbyint()'
+Description: 'Round off the given argument(value) to the nearest integral value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `cmath` header defines `nearbyint()` function that returns the nearby integral value of a given argument. It takes a single argument as a floating point value and returns the round off value of the argument as an integer.
```

Comment:
```suggestion
The **`nearbyint()`** function returns an integer that is closest to a given floating point argument.
```

With Docs entries, we definitely want to be descriptive but not too verbose. The opening sentences here should plainly state what the function returns in relation to the given argument(s).